db/seeds.d/60-host_reports_feature.rb

```diff
@@ -0,0 +1,2 @@
+f = Feature.where(name: 'HostReports').first_or_create
```

Comment:
If you introduce this feature (which I like), I think you should update this code to be `HostReports` only:
https://github.com/theforeman/foreman_host_reports/blob/af5fd1626d5d73b1b6024408ae4e706c9cb2ff99/app/models/host_report.rb#L48-L50